internal/util/copy.go

```diff
@@ -83,6 +99,45 @@ func CopyIndexFiles(logger *log.CmdLogger, src, dest string) error {
 	return nil
 }
 
+// skipFolder checks if the folder will need to be skipped during indexes copy.
+func skipFolder(logger *log.CmdLogger, indexSrcDir, src string) (bool, error) {
+	path, _ := filepath.Rel(indexSrcDir, src)
+
+	matchContainsPublicIndexFolder, _ := filepath.Match("index*", path)
+	matchContainsPrivateDataCollectionFolder, _ := filepath.Match("collections*", path)
+	matchPrivateDataCollectionFolder, _ := filepath.Match("collections/*", path)
+	matchPrivateDataCollectionIndexFolder, _ := filepath.Match("collections/*/indexes", path)
```

Comment:
I wonder if this pattern should be, `collections/*/indexes/*`

Reply:
`collections/*/indexes/*` will permit this `collections/*/indexes/anotherfolder` correct?